sympy/core/function.py

```diff
@@ -2276,9 +2276,13 @@ def doit(self, **hints):
                 # do Subs that aren't related to differentiation
                 undone2 = []
                 D = Dummy()
+                arg = e.args[0]
+                arg = repr(arg)
```

Comment:
Why is this call to `repr` here?

Reply:
Thanks for the feedback.
Used **repr** to convert the argument i.e. arg to string as **.find()** method takes a string as input, since e.args[0] returns an **Add** object.